### PR TITLE
Implement faster serial from the prawn projects.

### PR DESCRIPTION
**Pull request by dihm**
Faster serial comms are implemented.

Still need to implement a binary communication protocol for larger tables to greatly improve speed of programming.

### CODE REVIEW

**Comment by dihm** [comment]
Going to go ahead with this change as-is, and leave the bulk binary transfer to another PR. @carterturn, if you can confirm I didn't bork anything too badly, I'll go ahead and merge this.

File: dds-sweeper/dds-sweeper.c [hunk 1]
```diff
@@ -873,9 +859,9 @@ void loop() {
         int parsed = sscanf(readstring, "%*s %u %u %u", &type, &_timing);
```
Comment (carterturn) [inline dds-sweeper/dds-sweeper.c:859]: For some reason, the extra `%u` in the format string is now causing my Pico to freeze when the `mode` command is used.

**Review by carterturn — CHANGES_REQUESTED** [review]
Bug in the `mode` command is now causing issues.

The `readregs` command performs `printf`s from `ad9959.c`; this should also be switched to `fast_serial_printf`.

File: dds-sweeper/dds-sweeper.c (hunk 1 above)
Comment (dihm) [reply dds-sweeper/dds-sweeper.c:859]: That is odd. Not even sure why it's there. I'll remove it.

**Comment by dihm** [comment]
@carterturn I believe I've fixed the issues. My casual testing shows the device serial communication is functional.

**Review by carterturn — APPROVED** [review]
This passes all of my tests as well now.

**Pull request merged**